packages/docs/getting-started/running-a-validator.md

```diff
@@ -46,33 +66,227 @@ The code you'll see on this page is bash commands and their output.
+| CELO_VALIDATOR_GROUP_ADDRESS | The etherbase public address for the validation group                         |               |
+| CELO_VALIDATOR_ADDRESS       | The etherbase public address for the validator instance                       |               |
+| CELO_PROXY_ADDRESS           | The etherbase public address for the proxy instance                           |               |
+| CELO_VALIDATOR_POP           |                                                                               |               |
+| PROXY_ENODE                  | The ethereum node address for the validator                                   |               |
+| PROXY_IP                     | The proxy container internal IP address from docker pool address              |               |
+| ATTESTATION_KEY              | The etherbase private key for the account used in the attestation service     |               |
+| ATTESTATION_SERVICE_URL      | The URL to access the attestation service deployed                            |               |
+| METADATA_URL                 | The URL to access the metadata file for your attestation service              |               |
 
 If you are re-running these instructions, the Celo Docker image may have been updated, and it's important to get the latest version.
 
-Run:
+To run a complete validator it's necessary to execute the following components:
+
+- The valitor software
+- A Proxy that acts as an intermediary for the validator requests
+- The attestation service
+
+First we are going to setup the main environment variables related with the `Baklava` network. Run:
 
 ```bash
-docker pull us.gcr.io/celo-testnet/celo-node:alfajores`
+export CELO_NETWORK=baklava
+export CELO_IMAGE=us.gcr.io/celo-testnet/celo-node
+export NETWORK_ID=1101
 ```
 
-## **Create accounts**
+In all the commands we are going to see the `CELO_IMAGE` and `CELO_NETWORK` variables to refer to the right Docker image and network to use. Now we can get the Docker image:
 
-Create and cd into the directory where you want to store the data and any other files needed to run your node. You can name this whatever you’d like, but here’s a default you can use:
+```bash
+docker pull $CELO_IMAGE:$CELO_NETWORK
+```
+
+#### Create accounts
+
+At this point we need to create the accounts that will be used by the Validator and the Proxy. We create and cd into the directory where you want to store the data and any other files needed to run your node. You can name this whatever you’d like, but here’s a default you can use:
 
 ```bash
-mkdir celo-data-dir
+mkdir -p celo-data-dir/proxy celo-data-dir/validator
 cd celo-data-dir
 ```
 
+We are going to need to create 3 accounts, 2 for the validator and 1 for the Proxy.
+
+First we create three accounts, one for the Validator, one for Validator Group and the last one for the Proxy. You can get their addresses if you don’t already have them. If you already have some accounts, you can skip this step.
```

Comment:
"one for Validator Group" -> "one for the Validator Group"

Maybe make it clearer that "get their addresses" is in reference to later in this step-- "generate their addresses in this step" or something

Reply:
rephrased